### PR TITLE
[NEXT-6109] Add supportText to theme config

### PR DESCRIPTION
**Pull request by lx-wnk**
### 1. Why is this change necessary?
To provide and display the `supportText` inside the theme

### 2. What does this change do, exactly?
Add possibility to fill and display the `supportText`

### 3. Describe each step to reproduce the issue or behaviour.
Try to add supportText

### 4. Please link to the relevant issues (if any).
#812
Video: https://vimeo.com/418392136/0bd1eb2d46

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
  - Is this needed? There aren't any
- [x] I have squashed any insignificant commits
- [x] I have written or adjusted the documentation according to my changes
  - Due to the missing helpText-Field in the documentation I didn't adjust those
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by lx-wnk** [comment]
I'll fix the tests right now

File: src/Storefront/Migration/Migration1589443055AddThemeSupportText.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class Migration1589443055AddThemeSupportText extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1589443055;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeUpdate('ALTER TABLE `theme_translation` ADD `support_texts` json NULL AFTER `labels`;');
```
Comment (mitelg) [inline src/Storefront/Migration/Migration1589443055AddThemeSupportText.php:17]: why should a textfield be of type json?

Comment (mitelg) [reply src/Storefront/Migration/Migration1589443055AddThemeSupportText.php:17]: never mind. same logic as the other texts

**Comment by leichteckig** [comment]
QA approved, as soon as you remove the duplicates for testing purposes :+1:

**Comment by pantrtxp** [comment]
@jinnoflife can you please resolve your conflicts :)?

File: src/Administration/Resources/app/administration/test/app/component/form/__snapshots__/sw-radio-field.spec.js.snap [hunk 2]
```diff
@@ -109,6 +109,8 @@ exports[`components/form/sw-radio-field should renders correctly with children s
     </div>
   </div>
    
+  <!---->
```
Comment (tobiasberge) [inline src/Administration/Resources/app/administration/test/app/component/form/__snapshots__/sw-radio-field.spec.js.snap:112]: Is this wanted? 🤔

**Comment by lx-wnk** [comment]
@pantrtxp done
@leichteckig which duplicates do you mean?

**Comment by mitelg** [comment]
@jinnoflife please add the `fixes #123` hint to your commit message

**Review by tobiasberge — CHANGES_REQUESTED** [review]

**Review by tobiasberge — APPROVED** [review]

**Comment by mitelg** [comment]
@jinnoflife please rebase your branch :blush: there are merge conflicts

**Comment by pweyck** [comment]
Hi @jinnoflife, could you please rebase this onto master and resolve @tobiasberge's comment?

**Comment by tobiasberge** [comment]
@pweyck Sorry my bad this was auto generated by jest snapshot. It just looked weird.

**Pull request closed without merging**